openfisca_tunisia/tunisia_taxbenefitsystem.py

```diff
@@ -20,19 +20,20 @@ class TunisiaTaxBenefitSystem(TaxBenefitSystem):
     DEFAULT_DECOMP_FILE = decompositions.DEFAULT_DECOMP_FILE
     REFORMS_DIR = os.path.join(COUNTRY_DIR, 'reformes')
     REV_TYP = None
-    REVENUES_CATEGORIES = {
+    REVENUES_CATEGORIE_SALARIEEGORIES = {
         'imposable': ['sal'],
         }
 
     columns_name_tree_by_entity = datatrees.columns_name_tree_by_entity
 
     def __init__(self):
-        TaxBenefitSystem.__init__(self, entities.entities)
+        # We initialize our tax and benefit system with the general constructor
```

Comment:
Remplacer le commentaire par une indication sur pourquoi ceci est nécessaire ?

Reply:
Be my guest

Reply:
Sorry, rien à faire car plus simple qu'imaginé (l'écart découle d'un simple choix syntaxique).